onnx_tf/handlers/backend/non_zero.py

```diff
@@ -0,0 +1,22 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("NonZero")
+class NonZero(BackendHandler):
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
+    inp = kwargs["tensor_dict"][node.inputs[0]]
+    condition = tf.not_equal(inp, tf.zeros_like(inp))
+    nonzero_indices = tf.where(condition)
+    return [
```

Comment:
@fumihwh I cannot remember if there's any consequence for just returning tf.transpose(inp) instead of going through `make_tensor_from_onnx_node`.

Looks redundant here with the call to `make_tensor_from_onnx_node`.

Reply:
@tjingrant Yes, you're right. Just return [tf.transpose(inp)] is ok.

Reply:
@tjingrant May you please clarify more about the rule on when to call make_tensor_from_onnx_node and when can simply return the tf.some_operator.

Reply:
I think we should prefer whichever one is simpler.

For instance, the combination of attribute mapper and `make_tensor_from_onnx_node` can make handlers very concise when you just need to map the attributes and forward them to tensorflow function, for example: https://github.com/onnx/onnx-tensorflow/blob/8563315b3e32293f84c599346d73fbf0a1445187/onnx_tf/handlers/backend/cast.py#L22.